book/doctrine.rst

```diff
@@ -916,8 +924,81 @@ a ``name`` field and the associated getter and setter functions.
 Relationship Mapping Metadata
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
I felt like a more-detailed explanation of the difference between `OneToMany` and `ManyToOne` was warranted here, since we're dealing with readers who may not be familiar with those terms (and who may be confused by the fact that both terms can be used to describe the same relationship).